CHANGES.rst

```diff
@@ -1,3 +1,146 @@
+  This change was made to ensure that ``from urllib3 import request`` imported the top-level ``request()``
+  function instead of the ``urllib3.request`` module. (`#2269 <https://github.com/urllib3/urllib3/issues/2269>`__)
+- Removed support for SSLv3.0 from the ``urllib3.contrib.pyopenssl`` even when support is available from the compiled OpenSSL library. (`#2233 <https://github.com/urllib3/urllib3/issues/2233>`__)
+- Removed the deprecated ``urllib3.contrib.ntlmpool`` module. (`#2339 <https://github.com/urllib3/urllib3/issues/2339>`__)
+- Removed the list of default ciphers for OpenSSL 1.1.1+ and SecureTransport as their own defaults are already secure. (`#2082 <https://github.com/urllib3/urllib3/issues/2082>`__)
+
+Changed
+-------
+
+- Changed ``urllib3[brotli]`` extra to install Google's Brotli C bindings on CPython.
+  The Brotli CFFI bindings are still used on non-CPython implementations like PyPy.
+  This change was made to increase performance for CPython as CFFI is typically
+  is less performant than C extensions. (`#2099 <https://github.com/urllib3/urllib3/issues/2099>`__)
+- Changed default ``SSLContext.minimum_version`` to be ``TLSVersion.TLSv1_2`` in line with Python 3.10. (`#2373 <https://github.com/urllib3/urllib3/issues/2373>`__)
+- Changed ``urllib3.HTTPConnection.getresponse`` to return an instance of ``urllib3.HTTPResponse`` instead of ``http.client.HTTPResponse``. (`#2648 <https://github.com/urllib3/urllib3/issues/2648>`__)
+- Changed return type of ``HTTPResponse.getheaders()`` method to return a list of key-value tuples to match CPython. (`#1543 <https://github.com/urllib3/urllib3/issues/1543>`__)
+- Changed the error raised when connecting via HTTPS when the ``ssl`` module isn't available from ``SSLError`` to ``ImportError``. (`#2589 <https://github.com/urllib3/urllib3/issues/2589>`__)
+- Changed ``HTTPConnection.request()`` to always use lowercase chunk boundaries when sending requests with ``Transfer-Encoding: chunked``. (`#2515 <https://github.com/urllib3/urllib3/issues/2515>`__)
+- Changed `enforce_content_length` default to True, preventing silent data loss when reading streamed responses. (`#2514 <https://github.com/urllib3/urllib3/issues/2514>`__)
+- Changed internal implementation of ``HTTPHeaderDict`` to use ``dict`` instead of ``collections.OrderedDict`` for better performance. (`#2080 <https://github.com/urllib3/urllib3/issues/2080>`__)
+- Changed the ``urllib3.contrib.pyopenssl`` module to wrap ``OpenSSL.SSL.Error`` with ``ssl.SSLError`` in ``PyOpenSSLContext.load_cert_chain``. (`#2628 <https://github.com/urllib3/urllib3/issues/2628>`__)
+- Changed usage of the deprecated ``socket.error`` to ``OSError``. (`#2120 <https://github.com/urllib3/urllib3/issues/2120>`__)
+
+Deprecated
+----------
+
+- Deprecated ``urllib3.contrib.pyopenssl`` module which will be removed in future urllib3 v2.x release. (`#2691 <https://github.com/urllib3/urllib3/issues/2691>`__)
```

Comment:
This has been mentioned for 1.26.12, but may be worth mentioning here again

Reply:
Yeah I'll reiterate this one since it's being removed in v2.1.0.